.vib/opencart/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,40 @@
+const COMMAND_DELAY = 500;
+const BASE_URL = 'http://vmware-opencart.my';
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
+}
+
+// Due to a bug when using "hosts" in Cypress, we cannot set a "baseUrl" in the
+// cypress.json file. Workaround this by modifying the "visit" command to preprend
+// the base URL.
+//
+// Further details: https://github.com/cypress-io/cypress/issues/20647
+Cypress.Commands.overwrite('visit', (originalFn, url, options) => {
+  return originalFn(`${BASE_URL}${url}`, options);
+});
+
+Cypress.Commands.add(
+  'login',
+  (username = Cypress.env('username'), password = Cypress.env('password')) => {
+    cy.visit('/admin');
+    cy.get('.panel-title');
```

Comment:
I may be missing something but this seems like an unnecessary get, as you are already checking the login page is loaded with the following lines.

Reply:
Since the login form is the first point of contact right after the app is installed, I like to have an additional check for the login form before actually completing the form.
I also do this in many other apps. It's an anti-flake practice.